core/src/main/java/nl/nn/adapterframework/senders/XsltSender.java

```diff
@@ -172,7 +195,21 @@ public String sendMessage(String correlationID, String message, ParameterResolut
 //				log.debug(getLogPrefix()+" prc.inputsource ["+prc.getInputSource()+"]");
 //			}
 
-			stringResult = transformerPool.transform(inputMsg, parametervalues);
+			TransformerPool poolToUse = transformerPool;
+			
+			
+			if(!StringUtils.isEmpty(styleSheetNameSessionKey) && prc.getSession().get(styleSheetNameSessionKey) != null) {
```

Comment:
In plaats van `!StringUtils.isEmpty()` kan je ook `StringUtils.isNotEmpty()` gebruiken